l1-contracts/test/foundry/integration/deploy-scripts/script-out/output-deploy-l1.toml

```diff
@@ -0,0 +1,56 @@
+create2_factory_addr = "0x4e59b44847b379578588920cA78FbF26c0B4956C"
+create2_factory_salt = "0x00000000000000000000000000000000000000000000000000000000000000ff"
+deployer_addr = "0x7FA9385bE102ac3EAc297483Dd6233D62b3e1496"
+era_chain_id = 9
+force_deployments_data = "0x000000000000000000000000000000000000000000000000000000000000002000000000000000000000000000000000000000000000000000000000000000030000000000000000000000000000000000000000000000000000000000000060000000000000000000000000000000000000000000000000000000000000016000000000000000000000000000000000000000000000000000000000000002a044dab1170638e5acb1c036d7f2528bf6365021e89f5eb35fad0e98975382db5000000000000000000000000000000000000000000000000000000000000100020000000000000000000000000000000000000000000000000000000000000001000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000a00000000000000000000000000000000000000000000000000000000000000040000000000000000000000000000000000000000000000000000000000000010e00000000000000000000000081aa7970c51812dc3a010c7d01b50e0d17dc8ad9d3de98fb9db900fd0eb69c4e082f0a9b60872c04e318308d2128ee8ac82d634900000000000000000000000000000000000000000000000000000000000100030000000000000000000000000000000000000000000000000000000000000001000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000a00000000000000000000000000000000000000000000000000000000000000080000000000000000000000000000000000000000000000000000000000000010e000000000000000000000000000000000000000000000000000000000000000900000000000000000000000070997970c51812dc3a010c7d01b50e0d17dc79c800000000000000000000000000000000000000000000000000000000000000013927048defd1ff97c7b682da4c45b7f2f838b0ad9d56f6754033951c9e6f3e1800000000000000000000000000000000000000000000000000000000000100040000000000000000000000000000000000000000000000000000000000000001000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000a000000000000000000000000000000000000000000000000000000000000000c0000000000000000000000000000000000000000000000000000000000000010e00000000000000000000000081aa7970c51812dc3a010c7d01b50e0d17dc8ad9349c400bc5cef9910dcd7cd1328960b14d2c095f0e5d26c14f4f8a467160578500000000000000000000000070997970c51812dc3a010c7d01b50e0d17dc79c800000000000000000000000070997970c51812dc3a010c7d01b50e0d17dc79c80000000000000000000000000000000000000000000000000000000000000000"
```

Comment:
previously I put the forceDeployments data in script-config/force-deployments-data.toml, 
https://github.com/matter-labs/era-contracts/blob/709bd86061a13342399e33a02110cbb3677ddd77/l1-contracts/.env#L42
This was changed then right? I don't see the .env or the GenerateForceDeployments.s.sol script changed appropriately